### ISSUE TITLE
Database not marked as unsaved when KeePassXC-Browser connection is made

### ISSUE THREAD

**Issue by JakeTheSnake3p0** [smorks/keepassnatmsg#85]
For the last few days I've been wondering why I have had to keep reconnecting to my database every so often. I looked into the plugin's key storage and noticed that the keys were present after connecting, but missing when I restarted keepass. When creating the connection through the browser extension, the save icon on keepass never became active (just stayed grey). I decided to click save anyway, and that fixed the problem.

Basically there's a UI glitch where keepass doesn't visually mark the database as changed which would normally prompt the user through a colourful save icon and an asterisk to save the recently changed database.

**Comment by smorks** [comment]
i believe i've noticed this too. i'll look into it.
